WordPressAuthenticator/Unified Auth/View Related/Reusable Views/TextLinkButtonTableViewCell.swift

```diff
@@ -18,15 +18,20 @@ class TextLinkButtonTableViewCell: UITableViewCell {
 
     public var actionHandler: (() -> Void)?
 
+	override func awakeFromNib() {
+		super.awakeFromNib()
+
+		button.titleLabel?.font = UIFont.preferredFont(forTextStyle: .callout)
+		button.titleLabel?.adjustsFontForContentSizeCategory = true
+		button.titleLabel?.lineBreakMode = .byWordWrapping
```

Comment:
Just a thought here: since you're using a `xib`, it would be nice to set these at design time, although, for a `UIButton` is less obvious, since you don't have the 'adjustsFontForContentSizeCategory` readily available. You might still set the font and line break mode in there. You could also set `adjustsFontForContentSizeCategory` in there using `@ibdesignable` (which I don't think is worth) or trying to set that value as a user defined runtime attribute (but it would still be a runtime calculation). Again it's just a thought, there's nothing wrong in leaving as it is.

Reply:
Okie dokie, I moved setting the font and lineBreakMode to the Interface Builder. `ajustsFontForContentSizeCategory` is a variable in the `UIContentSizeCategoryAdjusting` protocol. Protocol variables don't seem to work with @IBInspectable or @IBDesignable, so I left it as-is.